CDC-Data-Reconciliation-Frontend/src/components/Report.jsx

```diff
@@ -446,7 +462,7 @@ export default function Report({ reportID }) {
               <button
                 type='button'
                 className='bg-blue-400 text-white px-5 py-2 rounded-md hover:bg-blue-600 flex flex-row items-center justify-around gap-2'
-                onClick={handleDownload}
+                onClick={handleResultsDownload}
```

Comment:
Good job on changing the function name as we now have multiple buttons on the page that can download data.